### PR TITLE
Restructure sys-mor tutorial

### PR DESCRIPTION
**Pull request by pmli**
This is still a work in progress. The idea is to add an LTI tutorial and for the existing BT tutorial to reuse the model from there. An IRKA tutorial could be added in a new PR

### CODE REVIEW

File: docs/source/tutorial_bt.rst [hunk 1]
```diff
@@ -273,20 +230,14 @@ and its Bode plot
 
     _ = err.bode_plot(w)
 
-We can compute the relative errors in :math:`\mathcal{H}_\infty` or
-:math:`\mathcal{H}_2` (or Hankel) norm
+We can compute the relative errors in different system norms.
```
Comment (sdrave) [inline docs/source/tutorial_bt.rst:233]:
```suggestion
Finally, we can compute the relative errors in different system norms.
```

**Review by sdrave — COMMENTED** [review]
Very nice!

File: docs/source/tutorial_bt.rst [hunk 2]
```diff
@@ -66,150 +18,115 @@ First, we do the necessary imports and some matplotlib style choices.
...
-:math:`\widetilde{B} = S^T B`,
-:math:`\widetilde{C} = C T`
-has Gramians :math:`\widetilde{P}` and :math:`\widetilde{Q}`, i.e., solutions to
-Lyapunov equations
+another realization can be obtained by replacing :math:`x(t)` with
+:math:`T \tilde{x}(t)` or by pre-multiplying the differential equation with an
+invertible matrix.
+In particular, there exist invertible transformation matrices
+:math:`T, S \in \mathbb{R}^{n \times n}` such that the realization with
+:math:`\tilde{E} = S^{\operatorname{T}} E T = I`,
```
Comment (lbalicki) [inline docs/source/tutorial_bt.rst:56]: This only holds if E is invertible right? Maybe it should be mentioned that we consider an LTI system with invertible E matrix at the beginning of the paragraph.

File: docs/source/tutorial_bt.rst [hunk 3]
```diff
@@ -66,150 +18,115 @@ First, we do the necessary imports and some matplotlib style choices.
...
-:math:`\widehat{B} = W^T B`,
-:math:`\widehat{C} = C V`,
-which satisfies the :math:`\mathcal{H}_\infty` (i.e., induced
-:math:`\mathcal{L}_2`) error bound
+:math:`\hat{E} = W^{\operatorname{T}} E V`,
+:math:`\hat{A} = W^{\operatorname{T}} A V`,
+:math:`\hat{B} = W^{\operatorname{T}} B`,
+:math:`\hat{C} = C V`.
+
+It is known that the reduced-order model is asymptotically stable if
```
Comment (lbalicki) [inline docs/source/tutorial_bt.rst:109]: I think it should also be mentioned that we assume the full order model to be asymptotically stable. Maybe we should do this when the LTI system is introduced in this tutorial.

File: docs/source/tutorial_lti_systems.rst [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+(i.e., :math:`u(t) \in \mathbb{R}` or :math:`y(t) \in \mathbb{R}`),
+
+.. math::
+
+    \lVert H \rVert_{\mathcal{H}_2}
+    =
+    \sup_{u \neq 0}
+    \frac{\lVert y \rVert_{\mathcal{L}_\infty}}{\lVert u \rVert_{\mathcal{L}_2}}.
+
+The computation of the :math:`\mathcal{H}_2` is based on the system Gramians
```
Comment (lbalicki) [inline docs/source/tutorial_lti_systems.rst:353]: :math:`\mathcal{H}_2` norm

**Review by lbalicki — COMMENTED** [review]
Agreed, very nice new structure!

**Comment by pmli** [comment]
Thanks for the comments, I made updates.

**Review by lbalicki — APPROVED** [review]

**Review by sdrave — APPROVED** [review]

**Pull request merged**